### DISCUSSION TITLE
"oh dude where's my sveltekit"

### DISCUSSION THREAD

**Discussion by howesteve** [sveltejs/kit#5982] [General]
After my latest ```pnpm update``` today. I think the title is self-explanatory.

**Comment by antony** [comment]
It isn't. Can you please update your discussion to elaborate?

**Comment by yonnic** [comment]
lot of things have changed and you need to run the migration script. It will fix most of the problems. Check out the announcements about the newest breaking changes.

**Reply by infovore** [reply]
and: [the documentation is up-to-date](https://kit.svelte.dev/docs/introduction). In particular, [the changes to routing](https://kit.svelte.dev/docs/routing) are all up-to-date there.

**Comment by trasherdk** [comment]
Pin version to latest compatible: `"@sveltejs/kit": "1.0.0-next.405"` until you are ready to dive in to the upgrade.
